src/main/scala/com/softwaremill/realworld/Main.scala

```diff
@@ -1,13 +1,14 @@
 package com.softwaremill.realworld
 
+import com.softwaremill.realworld.articles.ArticlesEndpoints
 import org.slf4j.LoggerFactory
 import sttp.tapir.server.interceptor.log.DefaultServerLog
 import sttp.tapir.server.ziohttp.{ZioHttpInterpreter, ZioHttpServerOptions}
 import zio.http.{HttpApp, Server, ServerConfig}
 import zio.{Console, ExitCode, Scope, Task, ZIO, ZIOAppArgs, ZIOAppDefault}
 
 object Main extends ZIOAppDefault:
-  val log = LoggerFactory.getLogger(ZioHttpInterpreter.getClass.getName)
+  private val log = LoggerFactory.getLogger(ZioHttpInterpreter.getClass.getName)
```

Comment:
that's probably a problem in adopt-tapir, but we should use the built-in ZIO logging (customising the backend)

Reply:
ah no, there's no logging in adopt-tapir - so it's just sth to change here